### DISCUSSION TITLE
Stuck installation sec onion 2.3.2

### DISCUSSION THREAD

**Discussion by Axtrus** [Security-Onion-Solutions/securityonion#1945] [Unsupported Versions] [answered]
In fact my installation of sec onion stuck several time on 79%(applying the hive Salt state), i don't have legitimate reason to explain that.
Can i have support this issue ?
How many time the setup of network and applications must do ?
must i have internet to perform it ?
I use the latest version 2.3.2

**Comment by weslambert** [comment]
On what kind of hardware are you attempting to install Security Onion?

**Comment by Axtrus** [comment]
This is the The Zotac Mini PC

Le mar. 17 nov. 2020 à 23:11, weslambert <notifications@github.com> a
écrit :

**Comment by dougburks** [comment, marked as answer]
> must i have internet to perform it ?

If you don't have Internet access, you can choose the Airgap option:
https://docs.securityonion.net/en/2.3/airgap.html
